### ISSUE TITLE
4.0.1 release hash does not match according to table

### ISSUE THREAD

**Issue by schwigri** [inspircd/inspircd#2100]
**Description**

Downloading the RHEL 9 rpm file, the sha256sum matches the one listed for Ubuntu 24.04 (e.g. it is `52a42a40ba70cd8f6fce8ea4fbc3cd8ece5001386bc2dbbb1a33bd5ccad8dae1`)

**Steps to reproduce the issue:**

1. Download inspircd-4.0.1-1.el9.x86_64.rpm
2. Run `sha256sum ./inspircd-4.0.1-1.el9.x86_64.rpm`
3. Result is `52a42a40ba70cd8f6fce8ea4fbc3cd8ece5001386bc2dbbb1a33bd5ccad8dae1`

**Describe the results you received:**

sha256sum result is listed for Ubuntu release

**Describe the results you expected:**

I expected sha256sum to match RHEL 9 release from table

**Additional information you deem important (e.g. issue happens only occasionally):**

I decided to test downloading ubuntu one and checking sha256sum and it matches the one listed for RHEL 9. I think they are just swapped in the table

**Output of `./bin/inspircd --version`:**

N/A it is not installed yet

**Comment by SadieCat** [comment]
Oops I think I got them the wrong way around when filling in the table.

I just updated it based on my local copy. Can you check it matches for you now?

**Comment by schwigri** [comment]
Yes the hashes match as expected now! Thanks for taking a look especially so quickly :)
